utils/check/handle_rpms.sh

```diff
@@ -43,7 +43,7 @@ sudo yum -y remove cart cart-debuginfo cart-devel cart-tests \
                    daos-tests CUnit dpdk fio fuse fuse-lib hwloc \
                    libabt0 libcmocka libfabric libfabric-devel libisa-l \
                    libisa-l_crypto libopa1 libpmem libpmemblk mercury ndctl\
-                   ompi openmpi3 openpa-devel \
```

Comment:
Not really related to this PR but I was looking at this file earlier, do you know where/how it is used at all?

Reply:
I don't. I just assumed it's used to setup build envs.

Reply:
@JohnMalmberg looks like you added these files, can you shed any light on how they're used?

Reply:
@brianjmurrell is the one that can probably shed more accurate light on this.
CI prebuilt images, some only in the past may have installed these packages which can interfere with CI tests.
Some of this is so that a prebuilt image is ready to go for developer use, some of these are in there for historical reasons, and we have not confirmed that we can remove them from the list yet, like "ompi" and "fuse*"

Reply:
I actually know nothing about what `utils/check/handle_rpms.sh` is or what it does.

Reply:
Thanks you all, I'll address this in a separate PR.